server/events/pull_closed_executor_test.go

```diff
@@ -201,7 +201,7 @@ func TestCleanUpLogStreaming(t *testing.T) {
 		// Create Log streaming resources
 		prjCmdOutput := make(chan *jobs.ProjectCmdOutputLine)
 		storageBackend := jobmocks.NewMockStorageBackend()
-		prjCmdOutHandler := jobs.NewAsyncProjectCommandOutputHandler(prjCmdOutput, logger, jobs.NewJobStore(storageBackend))
+		prjCmdOutHandler := jobs.NewAsyncProjectCommandOutputHandler(prjCmdOutput, logger, jobs.NewJobStore(storageBackend, nil))
```

Comment:
I'd prefer to not arbitrarily pass nil in for things, this could fail in the future if there's some modification to `NewjobStore` that calls a method on the scope object.

Reply:
used a test tally scope instead of nil